app/models/detainee.rb

```diff
@@ -10,4 +10,24 @@ def age
   def female?
     gender == 'female'
   end
+
+  ETHNICITY_CODES = [
+    'A1 - Asian or Asian British Indian',
```

Comment:
@doctorpod as a question,
would it be better to store the code (eg. `A2`) in the database

then map the code to the string (eg. `Asian or Asian British Pakistani`)

The codes are not likely to change,
but I think those strings are quite odd.

Reply:
Not really. If the codes or strings change it's jus ta case of editing the file. If they were stored in the DB we'd have to use a migration or datafix. I think it would be more involved.

Reply:
Maybe I'm misunderstanding,

but aren't you effectively storing in the database?

``` ruby
>> detainee.ethnicity = "A2 - Asian or British Pakistani"
>> detainee.save

>> puts detainee.ethnicity
A2 - Asian or British Pakistani
```

I'm suggesting it would have been better to store:

``` ruby
class Detainee # a presenter or whatever
  ETHNICITY_CODES = { "A2" =>  "A2 - Asian or British Pakistani"}
  def ethnicity_string
     ETHNICITY_CODES.fetch(ethnicity, ethnicity)
  end
end

>> detainee.ethnicity = "A2"
>> detainee.save

>> puts detainee.ethnicity
A2
>> puts detainee.ethnicity_string
A2 - Asian or British Pakistani
```